lib/processors/bundlers/moduleBundler.js

```diff
@@ -91,9 +91,9 @@ const log = require("@ui5/logger").getLogger("builder:processors:bundlers:module
  * @typedef {object} ModuleBundleOptions
  * @property {boolean} [optimize=true] Whether the module bundle gets minified
  * @property {boolean} [sourceMap=true] Whether to generate a source map file for the bundle
- * @property {boolean} [decorateBootstrapModule=false] If set to 'false', the module won't be decorated
+ * @property {boolean} [decorateBootstrapModule=false] If set to 'false', bootable bundles won't be decorated
```

Comment:
This should then also be updated in the docs: https://sap.github.io/ui5-tooling/pages/Configuration/

Reply:
or, for now, in the v3 migration guide?

Reply:
I think it's also applicable to the master, as only the text has been improved without changing the behavior, right?

Reply:
Done in https://github.com/SAP/ui5-tooling/pull/616